src/errors.ts

```diff
@@ -62,7 +61,7 @@ export function wrapApiError(maybeError: Error, message?: string): Error {
   }
   const {data, status} = res;
   let msg = `${prefix}API responded with status ${status}`;
-  const causeMsg = typeof data == 'string' ? data : data?.error?.message;
+  const causeMsg = typeof data == 'string' ? data : error.message;
```

Comment:
Take a look at changes I made in [clio PR for axios upgrade](https://github.com/faros-ai/clio/pull/658/files).  This might help here.

Reply:
thanks! will check it out

Reply:
referring from your PR, i did
`const res: AxiosResponse<any, any> | undefined = error.response;`
but i don't really get what's the difference between this and having `(data as any)?.error?.message`.
it seems to me that both assign `any` type to `data`.

Reply:
I believe the type for data was unknown before you added the type declaration.